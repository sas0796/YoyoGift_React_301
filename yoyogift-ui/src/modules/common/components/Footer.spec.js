import React from "react";
import { mount } from "enzyme";

import Footer from "./Footer";

describe("Footer Test Suite", () => {
  it("should render expected HTML", () => {
    //Mocks
    let classes = { root: { marginTop: "2px" } }
    expect(mount(<Footer classes={classes}/>).html()).toMatchSnapshot();
  });
});