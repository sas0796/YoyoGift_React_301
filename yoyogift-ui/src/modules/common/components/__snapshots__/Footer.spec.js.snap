// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Test Suite should render expected HTML 1`] = `"<div class=\\"Footer-root-1 [object Object]\\"><header class=\\"MuiPaper-root-11 MuiPaper-elevation4-17 MuiAppBar-root-2 MuiAppBar-positionStatic-6 MuiAppBar-colorDefault-8\\"><div class=\\"MuiToolbar-root-38 MuiToolbar-regular-40 MuiToolbar-gutters-39\\"><p style=\\"float: right;\\">Â©2019 YoYo, Inc. All Rights Reserved.</p></div></header></div>"`;
